cfgov/jinja2/v1/index.html

```diff
@@ -174,15 +174,15 @@ <h3 class="h1 m-card_heading">97% of consumers</h3>
                         <section class="content-l_col
                                         content-l_col-1-3">
                             {% for block in sections.value %}
-                                <div class="m-related-posts_list-container">
+                                <div class="m-related-posts_list-container m-list">
```

Comment:
So the `m-list_item` is used to make the links bold, so it is used, but you bring up a good point that perhaps it should not be. I can't change it here though without refactoring the component.